### DISCUSSION TITLE
Further plans on loop mode

### DISCUSSION THREAD

**Discussion by babielmam** [nolimits4web/swiper#6997] [General]
Hey @nolimits4web,

thank you for creating this awesome library.
I recently updated from version 5 all the way up to version 10. Along the way I noticed, that loop mode changed drastically (like a lot of others already pointed out). Specifically I have a lot of use cases, where prev/next slides are shown next to the active slide in a somewhat different visual manner (e.g. lower opacity). On the last or first slide prev/next slides are not shown however. The new loop mode either is buggy or it no longer supports such a use case - I am not yet sure.

I would like to stay on version 10 as it is always a good practice to keep up to date. As I haven't seen any comments regarding your plans with loop mode: Are there plans to bring back old loop mode or at least make it possible to use loop mode when slidesCount < slidesPerView * 2? If so, is there an estimation when it will be released? 

I know this is open-source and in no way I want to push you into something which you think is not suited for your library. But I need to know in order to make an educated guess, if it's better to stay forever on old v8 or stay on v10 and in the (possibly) short meantime deactivate loop mode.

Thanks for your input!

**Comment by nolimits4web** [comment]
> Are there plans to bring back old loop mode or at least make it possible to use loop mode when slidesCount < slidesPerView * 2?

No

You can still duplicate slides manually if it is not enough for you. 

>  On the last or first slide prev/next slides are not shown however

It dependes on your slidesPerView configuration

**Reply by babielmam** [reply]
Thanks for the input. 

Regarding the duplication of slides, I don't think this is optimal, because I don't know of a any way to tell Swiper that those slides are duplicates and e.g. not meant to be taken into consideration for pagination, etc.

I wrapped up a small example of what I am trying to achieve with both versions: 

- Swiper 8 https://codesandbox.io/p/sandbox/hungry-bell-f5rvcn 
- Swiper 10 https://codesandbox.io/p/sandbox/condescending-dream-ny6t3h 

How would you suggest to approach this use case in Swiper 10?
